codegen/src/main/scala/CodegenPlugin.scala

```diff
@@ -0,0 +1,62 @@
+package codegen
```

Comment:
Can you use `caliban.codegen` package and move the sources in a caliban/codegen folder?